### PR TITLE
Changes to DE plots code

### LINKED ISSUE #33: Create remove offset function

### PR DESCRIPTION
**Pull request by Alanocallaghan**
Same as #112 but without massive image files

Resolve #33

### CODE REVIEW

**Comment by Alanocallaghan** [comment]
Seems to me like the code in TestDE is very closely entangled with the code for offset correction, which is not ideal. 

This means that it is very hard to separate the code for generating offset and TestDE figures from the code used to perform the differential tests.

If you don't want a class for offset correction, it seems like the easiest solution might just be to remove the offset plot from TestDE, and add it as a separate function including a small amount of code duplication between the two.

**Comment by Alanocallaghan** [comment]
This also has the rather unfortunate side-effect that calculation of some measures may be repeated multiple times

File: R/AllGenerics.R [hunk 1]
```diff
@@ -1,6 +1,11 @@
 #########################################################################
 # New generic functions
+#' @export
 setGeneric("displayChainBASiCS", 
            function(object, ...) standardGeneric("displayChainBASiCS"))
+#' @export
 setGeneric("displaySummaryBASiCS", 
            function(object, ...) standardGeneric("displaySummaryBASiCS"))
+#' @export
```
Comment (catavallejos) [inline R/AllGenerics.R]: I did not copy this generic for now

**Comment by Alanocallaghan** [comment]
Closed for being :100:% bloat

**Pull request closed without merging**